### PR TITLE
[Merged by Bors] - Updated cargo lock + fix CI

### PR DESCRIPTION
**Pull request by rustonaut**
Both async-bindgen and xayn-ai had updates.

So this PR updates the `Cargo.lock` file.

### CODE REVIEW

**Review by x3ro — APPROVED** [review]
LGTM 🎉

**Comment by x3ro** [comment]
Code-wise this seems fine, though the dart parts seem to fail now. Is that a fluke, or caused by the change?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,6 +19,9 @@ concurrency:
 permissions:
   contents: read
 
+env:
```
Comment (acrrd) [inline .github/workflows/ci.yml:22]: Why we need to ass this?

Comment (rustonaut) [reply .github/workflows/ci.yml]: I removed it again, I tried to fix the CI but as the environment is not inherited by called workflows it doesn't work and the current CI is effectively broken/stuck until forcefully merging a PR with failing CI.

**Comment by rustonaut** [comment]
bors merge

**Comment by rustonaut** [comment]
bors merge

**Pull request closed without merging**